src/Unleash/Communication/UnleashApiClient.cs

```diff
@@ -211,29 +209,18 @@ public async Task<bool> RegisterClient(ClientRegistration registration, Cancella
             }
         }
 
-        public async Task<bool> SendMetrics(ThreadSafeMetricsBucket metrics, CancellationToken cancellationToken)
+        public async Task<bool> SendMetrics(MetricsBucket metrics, CancellationToken cancellationToken)
         {
-            if (metricsRequestsToSkip > metricsRequestsSkipped)
-            {
-                metricsRequestsSkipped++;
-                return false;
-            }
-
-            metricsRequestsSkipped = 0;
-
             const string requestUri = "client/metrics";
 
             var memoryStream = new MemoryStream();
 
-            using (metrics.StopCollectingMetrics(out var bucket))
+            jsonSerializer.Serialize(memoryStream, new
             {
-                jsonSerializer.Serialize(memoryStream, new ClientMetrics
-                {
-                    AppName = clientRequestHeaders.AppName,
-                    InstanceId = clientRequestHeaders.InstanceTag,
-                    Bucket = bucket
-                });
-            }
+                clientRequestHeaders.AppName,
```

Comment:
Don't really mind if you wanna keep it but this used to be a named field and now it's not. Intentional?

Reply:
So obviously not going to affect the outcome. But we should be consistent, we're using
`InstanceId = clientRequestHeaders.InstanceTag,` the row below so we should pick one and stick to it

Reply:
My editor complained about this one, due it to being redundant. I assume because in `AppName = clientRequestHeaders.AppName`, `AppName` can be deducted as the end property name, and as such it can be dropped.

That would not be the case with `InstanceId = clientRequestHeaders.InstanceTag` for example, since `InstanceId != InstanceTag`.

I'm OK with reverting this but I don't think it matters much 🤷

Reply:
> My editor complained about this one

> I don't think it matters much 🤷

Yuh, happy for you to leave it alone if it's a tool complaining

Reply:
Fair point, I'm not going to be too fussy about it either then :)